docs/build-on-linea/ethereum-differences.md

```diff
@@ -7,14 +7,15 @@ sidebar_position: 4
 
 ## EVM opcodes
 
-Linea uses the London version of the EVM. All EVM opcodes are managed identically to Ethereum London unless presented in the following table. Checkout [Ethereum's history](https://ethereum.org/en/history/) to learn about major milestones, forks, and updates to the blockchain.
+**Linea uses the London version of the EVM. All EVM opcodes are managed identically to Ethereum London unless presented in the following table. Checkout [Ethereum's history](https://ethereum.org/en/history/) to learn about major milestones, forks, and updates to the blockchain.**
 
 > ⚠️ At the moment we do not support the `eth_newFilter` and `eth_newBlockFilter` RPC calls.
 >
 
 | Opcode name | Ethereum | Linea |
 | --- | --- | --- |
 | `DIFFICULTY` / `PREVRANDAO` | Returns the RANDAO value from the previous block | Returns a fixed number |
+| `PUSH0` | Pushes the constant value 0 onto the stack | Linea supports Solidity compiler versions up to 0.8.19, matching Ethereum's London release. To resolve this issue, please recompile your contract using Solidity version 0.8.19 or lower. |
 
 _In Ethereum, this opcode was previously known as_ `DIFFICULTY`. _However, through the implementation of_ [EIP-4399](https://eips.ethereum.org/EIPS/eip-4399), _it was renamed to_ `PREVRANDAO`, _and its functionality changed due to the switch from PoW to PoS. Notably, it is a full 256-bit value that represents the randomness beacon output of the previous block. For more, see the 'Security Considerations' section of the above EIP._
```

Comment:
Should this line go inside the Linea heading in the table

Reply:
i dont think it should. the table is only for differences in opcode functionality, so it should behave exactly the same unless stated otherwise.

Reply:
I do find it weird that it says "this opcode" but is only specifically referring to difficulty and prevrandao. It worked before because the only opcode listed was prevrandao

Is there a way to format this so that it makes more sense? Such as adding it to the table under the "opcode" column?

Reply:
hi @cedrabner can you make edits here?

`this opcode` is ambiguous and only relates to `prevrandao` right?

Reply:
made a new column for the explanations. lmk if anytthing else needs to change